src/lib.rs

```diff
@@ -776,11 +776,10 @@ pub use zerocopy_derive::FromZeros;
 #[cfg_attr(doc_cfg, doc(cfg(feature = "derive")))]
 pub use zerocopy_derive::Immutable;
 
-/// Types which do not directly contain any [`UnsafeCell`]s.
+/// Types which are free from interior mutability.
 ///
-/// `T: Immutable` indicates that `T` does not contain any `UnsafeCell`s in its
-/// fields. `T` may still refer to types which contain `UnsafeCell`s: for
-/// example, `&UnsafeCell<T>` implements `Immutable`.
+/// `T: Immutable` indicates that `T` does not permit interior mutation, except
+/// by ownership or an exclusive borrow.
```

Comment:
```suggestion
/// by ownership or an exclusive (`&mut`) borrow.
```

In my experience, most casual Rust programmers aren't familiar with the "mutable references are actually exclusive references" perspective.

Reply:
Done